iron-form.html

```diff
@@ -343,7 +342,9 @@
     },
 
     _registerElement: function(e) {
-      var element = e.target;
+      // Get the actual element that fired the event
+      var element = Polymer.dom(e).path[0];
```

Comment:
Sure, but path is an array that's already set in the `event` object (I'm not calculating it), and I'm just indexing, so it's pretty much the same thing. I'll make the change, though :)